backend/app/views/spree/admin/shared/_address_form.html.erb

```diff
@@ -5,7 +5,7 @@
       <%= field_container (s_or_b + '_' + field), nil, class: ["form-group", "#{type}-row"] do %>
         <%= f.label :country_id, Spree.t(:country) %>
         <div id="<%= s_or_b %>country">
-          <%= f.collection_select :country_id, available_countries, :id, :name, {}, { class: 'select2 w-100' } %>
+          <%= f.collection_select :country_id, all_countries, :id, :name, {}, { class: 'select2 w-100' } %>
```

Comment:
Maybe leave this as `available_countries`, I'm thinking that Spree admin will be creating orders based on a per store basis, and in that context is it wiser to only offer what each store can offer?

If that is the intention, it might be a better flow when creating a new order, to first show the `Store` tab before the `Cart` tab, that way the first thing you have done is set a solid root for all things to come as the order is built up, This would be especially useful if Products get associated with certain Stores at some stage.

Reply:
@MatthewKennedy we had some situations with backend created orders for countries not supported in the storefront, so this allows for more flexibility to store owners. This change will be extracted to a separate PR, just a thing I've noticed going through addresses API :)

Reply:
That makes sense, give the admin more power.